File: 8.7.2025/Fibanocci.c
#include <stdio.h>
int main() {
    int n,t1 =0,t2=1,u;
    printf("Enter a number:\n");
    scanf("%d",&n);
    for(int i=1;i<=n;i++){
        printf("%d",t1);
        u = t1+t2;
        t1=t2;
        t2=u;
    }
    return 0;
}

File: 8.7.2025/Fibanocci.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        System.out.println("Enter a number:");
        int n = sc.nextInt();
        int t1=0;
        int t2 = 1;
        int u;
        System.out.printf("Fibanocci series is ");
        for(int i=1;i<=n;i++){
           System.out.printf("%d",t1);
           u = t1+t2;
           t1=t2;
           t2=u;
        }
        
    }
}
